Formula/safe.rb

```diff
@@ -0,0 +1,40 @@
+class Safe < Formula
+  desc "Vault CLI"
+  homepage "https://github.com/starkandwayne/safe"
+  url "https://github.com/starkandwayne/safe/archive/v0.8.0.tar.gz"
+  sha256 "0d592e7689e7c9d729f69671d3afa0b4400626fb025d56af5b857345d1310dbf"
+  depends_on "go" => :build
+  depends_on "gox" => :build
+  depends_on "vault"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["RELEASE_ROOT"] = (buildpath/"destdir")
+    ENV["TARGETS"] = "darwin/amd64"
+    ENV["VERSION"] = version
+
+    (buildpath/"src/github.com/starkandwayne/safe").install buildpath.children
+
+    (buildpath/"destdir").mkpath
+
+    cd "src/github.com/starkandwayne/safe" do
+      system "make", "release"
```

Comment:
This seems to be running `go get` even with `gox` already in the `$PATH`. We should either build this without using the `Makefile` or get upstream to handle the case where `gox` is already available.

Reply:
i'd be surprised if upstream wants to change their Makefile to accommodate us; i've rewritten the install block to take advantage of our bundled gox